versionutils/version.go

```diff
@@ -150,11 +182,21 @@ func ParseVersion(tag string) (*Version, error) {
 	if err != nil {
 		return nil, errors.Errorf("Patch version %s is not valid", versionParts[2])
 	}
+	rc := 0
+	if tagAndBuildMetadata != "" {
+		rcString := strings.TrimPrefix(tagAndBuildMetadata, "rc")
+		parsedRc, err := strconv.Atoi(rcString)
+		if err != nil {
+			return nil, errors.Errorf("Semver tag %s is not valid release candidate (must be 'rc' followed by int, i.e. 'rc5')", tagAndBuildMetadata)
```

Comment:
nit: should declare error at top of file according to solo style guides

also, i think you meant `e.g.` not `i.e.` in the error message